src/nvim/lua/vim.lua

```diff
@@ -286,6 +286,9 @@ local function __index(t, key)
   elseif key == 'lsp' then
     t.lsp = require('vim.lsp')
     return t.lsp
+  elseif key == 'fswatch' then
+    t.fswatch = require('vim.fswatch.fswatch')
```

Comment:
When you change the other thing, then you can just require `vim.fswatch` which is nice.

Reply:
I most certainly will. :)